make_8271_rom.c

```diff
@@ -490,7 +490,9 @@ main(int argc, const char* argv[]) {
   if ((size_t) write_ret != k_rom_size) {
     util_bail("can't write output rom");
   }
-  close(fd);
+  if (0 != close(fd)) {
```

Comment:
Thanks!
Can you change these to
close(fd) != 0   ?
it matches the style used elsewhere in beebjit.

Reply:
(and sorry these ROM programs are so terrible... they're a leftover from when beebjit was a toy; would have been better to require beebasm)

Reply:
Done.